android/assets/jsons/translations/French.properties

```diff
@@ -1108,7 +1108,7 @@ Sleep until healed = Dormir jusqu'à guérison
 Moving = En mouvement
 Set up = Installer
 Escort formation = Formation d'escorte
-Stop Escort formation = Fin de l'escorte
+Stop Escort formation = Stopper l'escorte
```

Comment:
It's for consistency: I also use "Stopper" for "Stopper le déplacement" (Stop Movement) and "Stopper l'automatisation" (Stop Automation).

In this button context, "Arrêter" sounds too _definitive_ (like "end something, turn off") whereas "Stopper" means it can be suspended and enabled right again if necessary.